tests/test_stackdocs_build.py

```diff
@@ -0,0 +1,29 @@
+"""Tests for the documenteer.stackdocs.build module (build-stack-docs
+executable).
+"""
+
+import os
+
+from documenteer.stackdocs import build
+
+
+def test_find_package_docs():
```

Comment:
I'm running these tests with pytest (literally just run `pytest` from the package root).